src/consensus/consensus.h

```diff
@@ -12,7 +12,13 @@
 static const unsigned int MAX_BLOCK_SERIALIZED_SIZE = 4000000;
 /** The maximum allowed weight for a block, see BIP 141 (network rule) */
 static const unsigned int MAX_BLOCK_WEIGHT = 4000000;
-/** The maximum allowed size for a block excluding witness data, in bytes (network rule) */
+/**
+ * The maximum allowed size for a block excluding witness data, in bytes (network rule).
+ * This parameter is largely superfluous because it is directly implied by the above block
+ *  weight limit, even when BIP 141 is not active.  It continues to exist for use in
+ *  various early tests that run before the witness data has been checked.
```

Comment:
After re-reading the diff I think I get it: the indentation is to clarify that the current line is part of the sentence of the line above.